docs/install/install.rst

```diff
@@ -8,6 +8,8 @@ Note: The version definition for the HIP runtime is different from CUDA. On an A
 ``hipRuntimeGerVersion`` function returns the HIP runtime version; on an NVIDIA platform, this function
 returns the CUDA runtime version.
 
+.. _install_prerequisites:
```

Comment:
where's the metadata?

Reply:
docs/install/build.rst was also missing metadata - added it.